README.md

```diff
@@ -125,10 +128,11 @@ a greater degree of control over buffer allocation and freeing.
 
 Given this backend accesses the NVIDIA driver directly, via NVIDIA's unstable API, this module is likely to break often with new versions of the kernel driver. If you encounter issues using this backend raise an issue and including logs generated by `NVD_LOG=1`.
 
-**Currently tested and verified generations**
+**Verified working generations**
 
 * Pascal (10XX)
-* Turing mobile (20XX)
+* Turing (20XX)
+* Ampere (30XX)
```

Comment:
Looks like we've also got a report of it working from an A4000, so we could add that here.